content/post/054-go-ipfs-0.4.22.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2019-08-06
+url: 93-go-ipfs-0.4.22
+title: go-ipfs 0.4.22 released
+author: Steven Allen
+---
+
+We're releasing a PATCH release of go-ipfs based on 0.4.21 containing some critical fixes.
+
```

Comment:
Let's release this post after the release process post so we can link to https://github.com/ipfs/blog/pull/266.